docs/apache-airflow/security/security_model.rst

```diff
@@ -97,10 +97,74 @@ capabilities authenticated users may have:
 
 For more information on the capabilities of authenticated UI users, see :doc:`/security/access-control`.
 
+Capabilities of DAG Authors
+---------------------------
+
+DAG authors are able to submit code - via Python files placed in the DAG_FOLDER - that will be executed
+in a number of circumstances. The code to execute is not verified nor checked nor sand-boxed by Airflow
```

Comment:
```suggestion
in a number of circumstances. The code to execute is neither verified, checked nor sand-boxed by Airflow
```
or could be simply "Airflow does not verify, check or sandbox the code to be executed".